### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jelveh**
This is the first version of CONTRIBUTING.md outlining basic instructions and rules for contributing to Puter. We want to keep it short, simple, and easy to follow. Everybody's welcome to comment on this. 

@AtkinsSJ @vineeth-vk11 @agentcobra @KernelDeimos @eltociear @lsileoni @sunjc826 @james-p-xu @LynanBreeze feel free to chime in based on your experience so far.

I'm going to keep this open for 24 hours awaiting comments.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (AtkinsSJ) [inline CONTRIBUTING.md]: Might be worth linking directly to the [new issue](https://github.com/HeyPuter/puter/issues/new) page instead? Not sure.

File: CONTRIBUTING.md
Comment (AtkinsSJ) [inline CONTRIBUTING.md]: We often want to know the information in the Puter about box, maybe mention that here too? Could even include a screenshot.

File: CONTRIBUTING.md
Comment (AtkinsSJ) [inline CONTRIBUTING.md]: I'd be tempted to make `good first issue` the link instead, it seems a bit more obvious to me.

File: CONTRIBUTING.md
Comment (AtkinsSJ) [inline CONTRIBUTING.md]: I think this and the CLA title should be `##`? They're part of contributing code.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Generally seems good to me. :+1:

**Comment by AtkinsSJ** [comment]
Oh and actually, I think you can specify a template that GitHub will use when creating new issues. It might be good to provide one with a few headings like "Expected", "Steps to reproduce", etc.

File: CONTRIBUTING.md
Comment (lsileoni) [inline CONTRIBUTING.md]: I suggest to either guide the contributor to apply the bug label here, or communicate to contributors, that maintainers will do that classification for them to disambiguate that flow.

Also if there will be a bug/issue template provided(as @AtkinsSJ) suggested, it should link to that as well. A good example would be [osu's bug issue template](https://www.github.com/ppy/osu/blob/master/.github/ISSUE_TEMPLATE/bug-issue.yml) (MIT licensed if that's of concern).

File: CONTRIBUTING.md
Comment (jelveh) [reply CONTRIBUTING.md]: That's a good idea. 

> Could even include a screenshot.

I need to provide generic information about providing their Puter version since the UI changes quite a bit.

File: CONTRIBUTING.md
Comment (jelveh) [reply CONTRIBUTING.md]: Makes sense.

File: CONTRIBUTING.md
Comment (jelveh) [reply CONTRIBUTING.md]: Great idea. We're going to label it for them for now so that we get the hang of the appropriate labels. I'm going to clarify this in the guide.

> Also if there will be a bug/issue template provided(as @AtkinsSJ) 

Thanks for the suggestions. Looking into it.

Comment (jelveh) [reply CONTRIBUTING.md]: Yep, good idea.

Comment (jelveh) [reply CONTRIBUTING.md]: [@AtkinsSJ @lsileoni] 
Decided to hold off on enabling issue templates for the next one month. I want us to get a feel of what our template should look like based on the first 50-100 issues filed. I put a reminder in my calendar to get back to this in a month.

File: CONTRIBUTING.md
Comment (AtkinsSJ) [inline CONTRIBUTING.md]: I don't think this twitter link is meant to be here? :laughing:

Comment (jelveh) [reply CONTRIBUTING.md]: wow!! life saver!!!

**Pull request merged**
